reference/lib/ReferenceOrderCombiner.sol

```diff
@@ -260,6 +264,10 @@ contract ReferenceOrderCombiner is
                 // Retrieve the offer item.
                 OfferItem memory offerItem = offer[j];
 
+                anyNativeOfferItems =
+                    anyNativeOfferItems ||
```

Comment:
I don't think it can be removed. It's in a loop, you'd overwrite the result every iteration - only the last iteration would be deciding. You want to check if _any_ of the offer items is native.

Reply:
I don't think it can be removed. It's in a loop, you'd overwrite the result every iteration - only the last iteration would be deciding. You want to check if _any_ of the offer items is native.

Reply:
oh yea my bad, didn't notice the loop in the GH diff